packages/react/src/useSubmissionState.js

```diff
@@ -190,17 +190,14 @@ const useSubmissionState = (
         )
 
         ;(
-          (
-            onSubmitRef
-            .current({
-              allValues,
-              registeredValues,
-            })
-          )
-          || (
-            Promise
-            .resolve()
-          )
+          Promise
+            .resolve(
+              onSubmitRef
+              .current({
+                allValues,
+                registeredValues,
+              })
+            )
```

Comment:
👍

Reply:
Since you asked about formatting, there's an extra tab where `resolve` is listed (that entire section). I like to have `.` on the same line so spacing defines scope rather than parens.

Honestly, I can go back and fix these little things on my own time if I feel it's necessary. I'd rather you feel more comfortable contributing to the project and not have to worry about formatting 😉.

---

Both Prettier and ESLint do the "tab on `.`" by default, so it's common practice even though it adds a ton of unnecessary tabbing and makes vertical code harder to follow. I also use line-wrap in my text editor which makes `.` tabbing even more problematic.

Eventually, I'll have a Prettier plugin to adjust formatting the way I like it, but that's not gonna be done for a while. I forked Prettier a while ago and was nearly done with it, but I need to release it as a plugin for a production release.

Reply:
Literally half of the time I spent on this PR was fighting with vscode about spacing and deactivating formatters 🤣 (and even then it liked adding extra spacing which I kept fighting to undo and then it would be redone when writing the next line and so on). Totally missed that extra indent, I'd noticed how your code style continued the next line on the same indent level. For more horizontal programming styles, an indent on the next line makes sense, because you have things like:
```typescript
Promise.resolve(...)
  .then(...)
  .catch(...)
```
so it allows you to quickly notice that it's a function chain block.

A prettier plugin would be awesome! Then I could have a git hook to use prettier with my own settings when downloading the code and with yours when uploading 😁.